.ci/pipeline_definitions

```diff
@@ -7,6 +7,10 @@ network-problem-detector:
         preprocess:
           'inject-commit-hash'
       publish:
+        oci-builder: docker-buildx
+        platforms:
+        - linux/amd64
+        - linux/arm64
```

Comment:
Is this the only change needed?
I see the `GOARCH` is hard-coded to `amd64` here https://github.com/gardener/network-problem-detector/blob/108e886b804d215dcac68091df8cad677691db48/Makefile#L47

Reply:
Good catch, let me check.

Reply:
TL;DR- Removing the `GOARCH=amd64` should be enough.

I think concourse starts a container with the specified platform, so it will start one container with golang image of amd64 arch and one container with golang image with arch arm64. They have default value for `GOARCH` of amd64 and arm64 respectively, so this should be enough, I think.